jetty-server/src/main/config/etc/jetty-ssl.xml

```diff
@@ -44,6 +44,7 @@
     <Call name="addCustomizer">
       <Arg>
         <New class="org.eclipse.jetty.server.SecureRequestCustomizer">
+          <Arg name="sniRequired" type="boolean"><Property name="jetty.ssl.sniRequired" default="true"/></Arg>
```

Comment:
Why does jetty.sslContext.sniRequired default to false but jetty.ssl.sniRequired default to true?